src/demo/helpers.js

```diff
@@ -9,6 +9,9 @@ export const backgroundPathForMode = mode => {
     imgName = 'underwater';
   }
 
+  // Temporarily show background for every mode.
```

Comment:
Why is this necessary?

Reply:
I wanted to see how UI elements looked on a background other than plain white, since I presume we will have some kind of background for each scene.